### DISCUSSION TITLE
on print sucess the printer keeps pausing

### DISCUSSION THREAD

**Discussion by sidneyns** [smartin015/continuousprint#188] [General]
Hello guys..
After the printer ends and it is runing the sucess script, the printer is pausing. The printer had already ended, but did not eject the part... it just went to pause 

this is the error
can someone help

OctoEverywhere used Smart Pause to protect your print while paused. Smart Pause turned off your hotend and retracted the z-axis away from the print.

When the printing is resumed, the hotend temp and z-axis state will automatically be restored before the print resumes.

**Comment by sidneyns** [comment]
in addition.. the Print Success script is running with other prints on queue... 

The files were added in the same time of the 1st file.

**Reply by smartin015** [reply]
`Print Success` is intended to run on every print in the queue. did you mean to use `Queue Finished`?

**Comment by smartin015** [comment]
Re: your main issue, I recommend filing a bug (rather than a discussion) and including a System Info bundle so I can look at the logs. 

You could also try asking OctoEverywhere maintainers, since it looks like your error is from their plugin.

**Comment by sidneyns** [comment]
Hi.. i uninstall and installed again and the issue was solved..

Em qui., 12 de jan. de 2023 19:22, Scott Martin ***@***.***>
escreveu:

**Reply by smartin015** [reply]
Ok, great
